src/amo/components/RatingManager/index.js

```diff
@@ -209,6 +210,7 @@ export class RatingManagerBase extends React.Component<InternalProps> {
           {/* eslint-enable react/no-danger */}
           <div className="RatingManager-ratingControl">
             {!this.isSignedIn() ? this.renderLogInToRate() : null}
+            Click to rate:
```

Comment:
This should be localized?

Reply:
> This should be localized?

yes, it should 😬 

> Starting to think that text should just not be here?

What do you mean - the text is in the figma spec, preceding the stars. 

Do you mean when logged out?  I understood the original intent of that blue banner in the logged out UI was to look like it covered the logged in UI - there is styling bug there that's squishing the text and forcing it to wrap, that I thought I fixed though.

Reply:
No you're right. It just looked like it might have been a placeholder left behind but not it shouuld be there just probably not when logged out. Myabe we should not render any of the start rating manager section then.

Reply:
Looks fine to me now 🤷